discover/faq.md

```diff
@@ -341,3 +341,142 @@ veFeeDistributor has a start\_time of 1663804800 (Thu Sep 22 2022 00:00:00).
 You can publish Fixed Price Assets to any chain that Ocean supports.
 
 </details>
+
+<details>
```

Comment:
Great start @Wanderclyffex 👍 One thing though, please add these questions in their respective section. You have here chain questions, general and so more. If you find it necessary, you can create new sections